src/actions/actionFinalize.tsx

```diff
@@ -125,13 +125,6 @@ export const actionFinalize = register({
           { x, y },
         );
       }
-
-      if (
-        !appState.activeTool.locked &&
-        appState.activeTool.type !== "freedraw"
-      ) {
-        appState.selectedElementIds[multiPointElement.id] = true;
-      }
     }
```

Comment:
Whas this removal intentional?

Reply:
I should have commented, but if you check the return value, we're doing the same thing there